### ISSUE TITLE
imresize error

### ISSUE THREAD

**Issue by since92m** [feichtenhofer/twostreamfusion#24]
hello, I got following error:
Expected input number 1, A, to be nonempty.
Error in imresize>parsePreMethodArgs (line 333)
validateattributes(A, {'numeric', 'logical'}, {'nonsparse', 'nonempty'}, mfilename, 'A', 1);
Error in imresize>parseInputs (line 248)
[params.A, params.map, params.scale, params.output_size] = ...
Error in imresize (line 141)
params = parseInputs(varargin{:});
Error in cnn_ucf101_get_im_flow_batch (line 248)
            imt =   imresize(gather(imt(dy:sz(1)+dy-1,dx:sz(2)+dx-1,:)), [opts.imageSize(1:2)]);
Error in getBatchWrapper_ucf101_rgbflow>getBatch (line 52)
im = cnn_ucf101_get_im_flow_batch(images, opts, ...
Error in getBatchWrapper_ucf101_rgbflow>@(imdb,batch,moreopts)getBatch(imdb,batch,opts,numThreads,trainopts,moreopts) (line 3)
fn = @(imdb,batch, moreopts) getBatch(imdb,batch,opts,numThreads, trainopts, moreopts) ;
Error in cnn_train_dag>process_epoch (line 306)
    [inputs] = state.getBatch(state.imdb, batch, moreopts) ;
Error in cnn_train_dag (line 108)
    s_train = process_epoch(net, state, opts, 'train');
Error in cnn_ucf101_fusion (line 349)
[info] = cnn_train_dag(net, imdb, fn, opts.train) ;

**Comment by Echotian** [comment]
I got the same error,did you solve it?

**Comment by since92** [comment]
No, I didn't solve it, I don't know how to debug, and I am trying using keras.

**Comment by since92m** [comment]
No, I didn't solve it, I don't know how to debug, and I am trying using keras.@Echotian

**Comment by Echotian** [comment]
I solved the problem，The path in  cnn_setup_environment.m can not contain Chinese characters.

**Comment by emigmo** [comment]
I also have the same problem, as follow : 
data/ucf101/jpegs_256/v_YoYo_g10_c05/frame000000.jpg: error ''
data/ucf101/jpegs_256/v_FieldHockeyPenalty_g12_c04/frame000000.jpg: error ''
data/ucf101/jpegs_256/v_HandstandPushups_g16_c06/frame000000.jpg: error ''
data/ucf101/jpegs_256/v_PlayingDaf_g12_c04/frame000000.jpg: error ''
data/ucf101/jpegs_256/v_UnevenBars_g16_c04/frame000000.jpg: error ''
data/ucf101/jpegs_256/v_HandstandPushups_g20_c04/frame000000.jpg: error ''
data/ucf101/jpegs_256/v_TrampolineJumping_g11_c01/frame000000.jpg: error ''
data/ucf101/jpegs_256/v_Rowing_g16_c01/frame000000.jpg: error ''
Warning: could not read image 'data/ucf101/jpegs_256/v_YoYo_g10_c05/frame000000.jpg' because '' 
> In cnn_ucf101_get_frame_batch (line 126)
  In getBatchWrapper_ucf101_imgs>getBatch (line 45)
  In getBatchWrapper_ucf101_imgs>@(imdb,batch,moreopts)getBatch(imdb,batch,opts,numThreads,trainopts,moreopts) (line 3)
  In cnn_train_dag>process_epoch (line 306)
  In cnn_train_dag (line 108)
  In cnn_ucf101_spatial (line 217) 
Warning: could not read image 'data/ucf101/jpegs_256/v_FieldHockeyPenalty_g12_c04/frame000000.jpg' because '' 
> In cnn_ucf101_get_frame_batch (line 126)
  In getBatchWrapper_ucf101_imgs>getBatch (line 45)
  In getBatchWrapper_ucf101_imgs>@(imdb,batch,moreopts)getBatch(imdb,batch,opts,numThreads,trainopts,moreopts) (line 3)
  In cnn_train_dag>process_epoch (line 306)
  In cnn_train_dag (line 108)
  In cnn_ucf101_spatial (line 217) 
Warning: could not read image 'data/ucf101/jpegs_256/v_HandstandPushups_g16_c06/frame000000.jpg' because '' 
> In cnn_ucf101_get_frame_batch (line 126)
  In getBatchWrapper_ucf101_imgs>getBatch (line 45)
  In getBatchWrapper_ucf101_imgs>@(imdb,batch,moreopts)getBatch(imdb,batch,opts,numThreads,trainopts,moreopts) (line 3)
  In cnn_train_dag>process_epoch (line 306)
  In cnn_train_dag (line 108)
  In cnn_ucf101_spatial (line 217) 
Warning: could not read image 'data/ucf101/jpegs_256/v_PlayingDaf_g12_c04/frame000000.jpg' because '' 
> In cnn_ucf101_get_frame_batch (line 126)
  In getBatchWrapper_ucf101_imgs>getBatch (line 45)
  In getBatchWrapper_ucf101_imgs>@(imdb,batch,moreopts)getBatch(imdb,batch,opts,numThreads,trainopts,moreopts) (line 3)
  In cnn_train_dag>process_epoch (line 306)
  In cnn_train_dag (line 108)
  In cnn_ucf101_spatial (line 217) 
Warning: could not read image 'data/ucf101/jpegs_256/v_UnevenBars_g16_c04/frame000000.jpg' because '' 
> In cnn_ucf101_get_frame_batch (line 126)
  In getBatchWrapper_ucf101_imgs>getBatch (line 45)
  In getBatchWrapper_ucf101_imgs>@(imdb,batch,moreopts)getBatch(imdb,batch,opts,numThreads,trainopts,moreopts) (line 3)
  In cnn_train_dag>process_epoch (line 306)
  In cnn_train_dag (line 108)
  In cnn_ucf101_spatial (line 217) 
Warning: could not read image 'data/ucf101/jpegs_256/v_HandstandPushups_g20_c04/frame000000.jpg' because '' 
> In cnn_ucf101_get_frame_batch (line 126)
  In getBatchWrapper_ucf101_imgs>getBatch (line 45)
  In getBatchWrapper_ucf101_imgs>@(imdb,batch,moreopts)getBatch(imdb,batch,opts,numThreads,trainopts,moreopts) (line 3)
  In cnn_train_dag>process_epoch (line 306)
  In cnn_train_dag (line 108)
  In cnn_ucf101_spatial (line 217) 
Warning: could not read image 'data/ucf101/jpegs_256/v_TrampolineJumping_g11_c01/frame000000.jpg' because '' 
> In cnn_ucf101_get_frame_batch (line 126)
  In getBatchWrapper_ucf101_imgs>getBatch (line 45)
  In getBatchWrapper_ucf101_imgs>@(imdb,batch,moreopts)getBatch(imdb,batch,opts,numThreads,trainopts,moreopts) (line 3)
  In cnn_train_dag>process_epoch (line 306)
  In cnn_train_dag (line 108)
  In cnn_ucf101_spatial (line 217) 
Warning: could not read image 'data/ucf101/jpegs_256/v_Rowing_g16_c01/frame000000.jpg' because '' 
> In cnn_ucf101_get_frame_batch (line 126)
  In getBatchWrapper_ucf101_imgs>getBatch (line 45)
  In getBatchWrapper_ucf101_imgs>@(imdb,batch,moreopts)getBatch(imdb,batch,opts,numThreads,trainopts,moreopts) (line 3)
  In cnn_train_dag>process_epoch (line 306)
  In cnn_train_dag (line 108)
  In cnn_ucf101_spatial (line 217) 
Error using imresize
Expected input number 1, A, to be nonempty.

Error in imresize>parsePreMethodArgs (line 333)
validateattributes(A, {'numeric', 'logical'}, {'nonsparse', 'nonempty'}, mfilename, 'A', 1);

Error in imresize>parseInputs (line 248)
[params.A, params.map, params.scale, params.output_size] = ...

Error in imresize (line 141)
params = parseInputs(varargin{:});

Error in cnn_ucf101_get_frame_batch (line 261)
            imt =   imresize(gather(imt(dy:sz(1)+dy-1,dx:sz(2)+dx-1,:)), opts.imageSize(1:2));

Error in getBatchWrapper_ucf101_imgs>getBatch (line 45)
im = cnn_ucf101_get_frame_batch(images, opts, ...

Error in getBatchWrapper_ucf101_imgs>@(imdb,batch,moreopts)getBatch(imdb,batch,opts,numThreads,trainopts,moreopts) (line 3)
fn = @(imdb, batch, moreopts) getBatch(imdb,batch,opts,numThreads, trainopts, moreopts) ;

Error in cnn_train_dag>process_epoch (line 306)
    [inputs] = state.getBatch(state.imdb, batch, moreopts) ;

Error in cnn_train_dag (line 108)
    s_train = process_epoch(net, state, opts, 'train');

Error in cnn_ucf101_spatial (line 217)
[info] = cnn_train_dag(net, imdb, fn, opts.train) ;
-------------------------------------------------------------------------------------------------------------------------
The dataset haven't   data/ucf101/jpegs_256/v_YoYo_g10_c05/frame000000.jpg
all the image is starting from frame000001.jpg, so >?

**Comment by since92m** [comment]
Thanks for your reply，but question is still there@Echotian

**Comment by since92x** [comment]
这一点我到没注意，一方面数据集没有了，另一方面尝试keras，good luck @emigmo

**Comment by emigmo** [comment]
The work will be very strong if rewrite the twostreamfusion framework using keras.
I just want to see the result of the  twostreamfusion, and I just want to know how he fusion the two stream in detail.

**Comment by JasmineeYang** [comment]
@emigmo @since92m @Echotian Hello ,I want to know if you have run the code successfully?For the reason that '/media/yangyang/DATA/learn/DP codes/ucf101-img-vgg16-split1-dr0.85.mat' is not exist ,so I changed the code just as follows:
% opts.modelA = fullfile(opts.modelPath, [opts.dataSet '-img-vgg16-split' num2str(opts.nSplit) '-dr0.85.mat']) ;
opts.modelA = fullfile(opts.modelPath, [opts.dataSet '-img-vgg16-split' num2str(opts.nSplit) .mat']) ;
yeah, I just throw away '-dr0.85',but other problems appear:
下标索引必须为正整数类型或逻辑类型。

出错 cnn_ucf101_get_im_flow_batch (line 95)
          last_frame = min(frameSamples(end), max(floor(nFrames - nStack/4 - opts.nFrameStack,nStack/4 )));

出错 getBatchWrapper_ucf101_rgbflow>getBatch (line 52)
im = cnn_ucf101_get_im_flow_batch(images, opts, ...

出错 getBatchWrapper_ucf101_rgbflow>@(imdb,batch,moreopts)getBatch(imdb,batch,opts,numThreads,trainopts,moreopts) (line 3)
fn = @(imdb,batch, moreopts) getBatch(imdb,batch,opts,numThreads, trainopts, moreopts) ;

出错 cnn_train_dag>process_epoch (line 306)
    [inputs] = state.getBatch(state.imdb, batch, moreopts) ;

出错 cnn_train_dag (line 108)
    s_train = process_epoch(net, state, opts, 'train');

出错 cnn_ucf101_fusion (line 353)
[info] = cnn_train_dag(net, imdb, fn, opts.train) ;
 I want to know where you modified?Did you run the code out successfully?
I hope to receive your reply. Thanks sincerely

**Comment by xiaollz** [comment]
@JasmineeYang I‘ve got the same error. Did you find the solution.
Thanks!

**Comment by ml930310** [comment]
@JasmineeYang Did you find the solution .Thanks!

**Comment by ml930310** [comment]
@since92m I‘ve got the same error .did you find the solution.

**Comment by ml930310** [comment]
@JasmineeYang 你可能是添加的路径不对。

**Comment by xiaollz** [comment]
I also solved the issue, and it is the problem with path.

**Comment by ml930310** [comment]
hello ,everyone.I I encountered a problem.
error using vl_nnconv
Out of memory on device. To view more detail about available memory on the GPU, use 'gpuDevice()'. If the problem persists, reset the GPU by calling 'gpuDevice(1)'.
Do you have solved?

**Comment by liwang621** [comment]
Hello，what does 66 lines'  "input_flow" play a role in the function of getBatchWrapper_ucf101_imgs ?

**Comment by lvyali** [comment]
> I also solved the issue, and it is the problem with path.

could you tell me the exact path you use in your codes?

**Comment by Haiyu12716** [comment]
> I also solved the issue, and it is the problem with the path.

can you tell me how to solve this issue in detail? thank you
